### ISSUE TITLE
Crash when using german locales

### ISSUE THREAD

**Issue by DerMaxPower** [eneshecan/whatsapp-for-linux#84]
Using Manjaro KDE I get this output when trying it to start from terminal when using german locales:
```
(process:3672): Gtk-WARNING **: 13:58:05.861: Locale not supported by C library.
        Using the fallback 'C' locale.

[Other warnings omitted due to being not relevant for this]

terminate called after throwing an instance of 'std::runtime_error'
  what():  locale::facet::_S_create_c_locale name not valid
Aborted (core dumped)
```
With en_US locales everything works fine.

**Comment by eneshecan** [comment]
Hi @DerMaxPower, thanks for reporting. Could you please tell me what you get from `locale -a` from terminal?

**Comment by eneshecan** [comment]
I should anyways handle the exception while trying to get the system locale for not crashing.

**Comment by eneshecan** [comment]
Fixed with 8ebc388
